partials/tabs/home.html

```diff
@@ -33,6 +33,11 @@ <h5 class="light">simply install coala</h5>
                     </p>
                 </div>
                 <div class="col m6 s12">
+                    <p>In order to use coala, you should have
+                        <a href="https://www.python.org/downloads/">Python >= 3.4</a> and
+                        <a href="https://pip.pypa.io/en/stable/installing/">pip >= 8</a> Installed
```

Comment:
Use `&lteq` instead of `>=`, at both the places ;)

Reply:
Actually, `&gteq` 😛

Reply:
yea, my bad :P

Reply:
I think it should be &ge. &gteq is showing `>eq`

Reply:
google it ;p